docs/uppmax.md

```diff
@@ -8,30 +8,32 @@ We have a Slack channel dedicated to UPPMAX users on the nf-core Slack: [https:/
 
 ## Using the UPPMAX config profile
 
+Before running the pipeline you will need to either install Nextflow or load it using the environment module system.
```

Comment:
```suggestion
Before running the pipeline you will need to either install Nextflow or load it using the environment module system (this can be done with e.g. `module load bioinfo-tools Nextflow/<VERSION>` where `VERSION` is e.g. `20.10`).
```